File: Java/program42.java

import java.util.Scanner;

class Display
{
    public void  Divisible(int iNo1 , int iNo2)
    {   
        int iCnt = 0;
        if(iNo1 % iNo2 ==0)
        {

            System.out.println("True");
        }
        else
        {
            System.out.println("False");
        }
}

    }

public class program42 {
    public static void main (String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        int iValue2 = 0;

        System.out.println("Please enter frequency : \n");
        iValue = sobj.nextInt();

        System.out.println("Please enter frequency : \n");
        iValue2 = sobj.nextInt();

        Display dobj = new Display();
        dobj.Divisible(iValue , iValue2);

    }
    
}

File: Python/program42.py
def display(i_no):
    # Time Complexity : O(N)
    for i_cnt in range(1, i_no + 1):
        if (i_cnt % 2) == 0:
            print(i_cnt, end="\t")
    print()

if __name__ == "__main__":
    i_value = int(input("Please enter frequency : \n"))
    display(i_value)